go.mod

```diff
@@ -3,18 +3,19 @@ module github.com/Scalingo/terraform-provider-scalingo
 go 1.17
 
 require (
-	github.com/Scalingo/go-scalingo v1.5.4-0.20201006151217-91760b279c98
+	github.com/Scalingo/go-scalingo/v4 v4.16.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.4.3
```

Comment:
The latest versino of this dependency is 2.17.0 (https://github.com/hashicorp/terraform-plugin-sdk/releases). Maybe use the last one?